lib/include/tick/base_model/model_lipschitz.h

```diff
@@ -24,14 +28,15 @@ class DLL_PUBLIC ModelLipschitz : public virtual Model {
   //! True if the mean of lipschitz constants is already computed
   bool ready_lip_mean;
 
-  //! All Lipschitz constants
-  ArrayDouble lip_consts;
-
   //! Average and maximum Lipschitz constants
-  double lip_mean, lip_max;
+  T lip_mean, lip_max;
+
+  //! All Lipschitz constants
+  Array<T> lip_consts;
 
  public:
-  ModelLipschitz();
+  TModelLipschitz();
+  virtual ~TModelLipschitz() {}
```

Comment:
Is this empty destructor needed now? Is it just cleaner to add an empty one?

Reply:
virtual destructors are sometimes necessary - and are good practice for base classes

https://stackoverflow.com/a/461224/795574

Reply:
Thanks!